configure.ac

```diff
@@ -1412,6 +1412,18 @@ if test "$cross_compiling" = yes; then
 	RUNSHARED=
 fi
 
+AC_ARG_VAR([HOSTRUNNER], [Program to run CPython for the host platform])
+AC_MSG_CHECKING([HOSTRUNNER])
+if test -z "$HOSTRUNNER"
+then
+  AS_CASE([$ac_sys_system/$ac_sys_emscripten_target],
+    [Emscripten/node*], [HOSTRUNNER='node'],
```

Comment:
```suggestion
    [Emscripten/node*], [
      AS_VAR_IF([enable_wasm_pthreads], [yes], [
        HOSTRUNNER='node --experimental-wasm-threads --experimental-wasm-bulk-memory'
      ], [
        HOSTRUNNER='node'
      ])
    ],
```

Reply:
Thanks! I was going to do this earlier but ended up in a meeting.